gui/mozregui/report.py

```diff
@@ -138,17 +178,20 @@ def step_build_found(self, bisection, build_infos):
             # and add the new step with build_infos
             item = StepItem()
             item.state_text = 'Downloading'
+            item.downloading = True
             item.data.update(build_infos)
             self.append_item(item)
         else:
             # previous item is a step, just update it
             last_item.data.update(build_infos)
+            last_item.downloading = True
```

Comment:
hm, this seems strange. The first upper "item.downloading = True" is not sufficient ?